test/integration/tox.ini

```diff
@@ -0,0 +1,14 @@
+[tox]
```

Comment:
It's good to keep ansible in `requirements.txt`, because for a python developer that's the fist place she would look into and setup runtime requirements in a virtual env (unrelated to tox).

The mismatch is most likely related to how pip works so far... I remember seeing an open issue to allow it to resolve required versions when two sources ask for different versions. The workaround is to call pip twice.